ranker/labelled_ranker.py

```diff
@@ -0,0 +1,118 @@
+import json, requests
+import simplejson
+import json
+
+filters=['easy','medium','hard','bug']
+stats=['additions','deletions','commits']
+scoring=dict(zip(filters+stats,[20,50,100,10]+[1,1,5]))
+users={}
+def init_user(login):
+    users[login]=dict(zip(filters+stats,len(filters+stats)*[0]))
+
+def req(get,params={}):
+    """Function to send HTTP Requests"""
+    params['access_token']='d0fec873c0a68b8e8b29b2403914a41f59479dc5'
```

Comment:
Making Access Tokens public is never a good idea. Use an environment variable or pass a command line argument.